rho/utilities.py

```diff
@@ -37,6 +37,9 @@
 ANSIBLE_LOG_PATH = os.path.join(DATA_DIR, 'ansible_log')
 SCAN_LOG_PATH = os.path.join(DATA_DIR, 'scan_log')
 
+PROFILE_HOSTS_SUFIX = '_hosts.yml'
```

Comment:
Should I go ahead and refactor the other files on this PR or send another one later?

Reply:
Lets do individual PRs